skimage/_shared/tests/test_safe_as_int.py

```diff
@@ -1,21 +1,31 @@
 import numpy as np
+import pytest
 from skimage._shared.utils import safe_as_int
 
 
 def test_int_cast_not_possible():
-    np.testing.assert_raises(ValueError, safe_as_int, 7.1)
-    np.testing.assert_raises(ValueError, safe_as_int, [7.1, 0.9])
-    np.testing.assert_raises(ValueError, safe_as_int, np.r_[7.1, 0.9])
-    np.testing.assert_raises(ValueError, safe_as_int, (7.1, 0.9))
-    np.testing.assert_raises(ValueError, safe_as_int, ((3,   4,   1),
-                                                       (2, 7.6, 289)))
-
-    np.testing.assert_raises(ValueError, safe_as_int, 7.1, 0.09)
-    np.testing.assert_raises(ValueError, safe_as_int, [7.1, 0.9], 0.09)
-    np.testing.assert_raises(ValueError, safe_as_int, np.r_[7.1, 0.9], 0.09)
-    np.testing.assert_raises(ValueError, safe_as_int, (7.1, 0.9), 0.09)
-    np.testing.assert_raises(ValueError, safe_as_int, ((3,   4,   1),
-                                                       (2, 7.6, 289)), 0.25)
+    with pytest.raises(ValueError):
```

Comment:
I'd be very surprised, as I expect the context manager to catch one and only one ValueError.
I can test.